### PR TITLE
Fix eigen3 bld

### PR DESCRIPTION
**Pull request by gonuke**
Fix the MOAB build in the docker image to use Eigen3 and no FORTRAN support

### CODE REVIEW

**Comment by bam241** [comment]
I'll run the scripts with the --no-fortran option (just to make sure it is working...)

**Comment by bam241** [comment]
removing --enableeigen works fine on my side :)

**Comment by bam241** [comment]
seems to be working.
[install.log](https://github.com/svalinn/DAGMC/files/4775834/install.log)

**Review by bam241 — APPROVED** [review]
sorry for missing it the first time.

LGTM!

File: news/PR-0684.rst [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+**Changed:** 
+* turn off fortran support in MOAB build
+
+**Deprecated:** None
+
+**Removed:** None
+
+**Fixed:** 
+* remove bad flag for Eigen3 n MOAB build
```
Comment (bam241) [inline news/PR-0684.rst]:
```suggestion
* remove bad flag for Eigen3 in MOAB build
```

**Comment by bam241** [comment]
rdy to merge this.
just want to let some time for other people to review :) (as I messed up the first time :p )

**Review by pshriwise — APPROVED** [review]
This looks good to me! A thank you for both this and #683 to remove our Fortran dependency in CI @gonuke.

**Comment by pshriwise** [comment]
This one has been open for some time and is a straightforward set of changes. Thanks @gonuke!

Merging...

**Pull request merged**